### ISSUE TITLE
refGenomes.len file location?

### ISSUE THREAD

**Issue by MatthewWolff** [sstevens2/mapMetasVsRefs#2]
All references to refGenomes.len indicate it should be in the main directory,  but there aren't any refGenome/*.len files being generated to create the refGenomes.len file, and thus it's always 0 bytes

**Comment by sstevens2** [comment]
I think this is probably because you are using fastq files where `scripts/countbasesFNA.py` runs on fasta files.  I need to add in an option for these two filetypes.  Something like..
```
find . -name '*.fastq' | xargs -t -P 4 -n 1 awk '{s++}END{print FILENAME,s/4}' > metaReads.txt
```
for fastq files.

@MatthewWolff If you'd like to add a check and this alternate option for fastq files, feel free to make a PR.  I'm pretty busy right now so I won't be able to get to it for a couple weeks probably.

**Comment by sstevens2** [comment]
Oh I think I wasn't thinking clearly here.  This is genome lengths not metagenome lengths like I was thinking.  What extensions do your reference genomes have?

**Comment by sstevens2** [comment]
Think this should be fixed as of commit a6c13ce to use files ending in fa, fna or fasta
